openshift/go.mod

```diff
@@ -1,5 +1,10 @@
-module _
+module github.com/operator-framework/ansible-operator-plugins/openshift
```

Comment:
The module is updated so that the internal `version` package can be imported in `openshift/version.go` to get the upstream version.

Reply:
Hmm, is there any alternative way to get the version? I'm not sure if this is a recommended practice to copy or vendor packages in this way. I saw the root Makefile is having 
```
export IMAGE_VERSION = v1.35.0
```
can we use this somehow?

Reply:
We don't need to update the module on every rebase as it is using the variable `ImageVersion` and will get the correct version. However, I feel your solution is simple and clean. Will update the PR.